utils/log-target.js

```diff
@@ -132,3 +131,4 @@ module.exports = class LoggingTarget {
     return throttleRate;
   }
 };
+module.exports.humanRtv = humanRtv;
```

Comment:
What's the needed for?

Reply:
I have no recollection of adding this. Removed.